index.html

```diff
@@ -1156,15 +1161,46 @@ <h2>Extensibility</h2>
         </p>
 
         <p>
-Implementations MUST produce an error when an extension JSON-LD Context
-overrides the expanded URL for a term specified in the base JSON-LD Context
-(<code>https://w3id.org/credentials/v1</code>). To avoid the possibility of
-accidentally overriding terms, developers are urged to scope their extensions.
-For example, the following extension scopes the new <code>favoriteFood</code>
-term so that it may only be used within the <code>claim</code> property:
+Developers are urged to ensure that extension JSON-LD Contexts are highly
+available. Implementations that cannot fetch a context will produce an error.
+Strategies for ensuring that extension JSON-LD Contexts are always available
+include using content-addressed URLs for contexts, bundling context documents
+with implementations, or enabling aggressive caching of contexts.
         </p>
 
-        <pre class="example nohighlight" title="Scoping terms in a JSON-LD Context">
+        <section>
+          <h3>Semantic Interoperability</h3>
+
+          <p>
+This specification endeavors to enable both the JSON and JSON-LD syntaxes
+to be semantically compatible with one another without the JSON implementations
+needing to process the documents as JSON-LD. In order to achieve this, the
+specification creates the following additional restrictions on both syntaxes:
+          </p>
+
+          <ul>
+            <li>
+JSON-based processors MUST process the <code>@context</code> property, ensuring
+the expected values exist in the expected order for the <code>type</code> of
+<a>credential</a> that they are processing. The expected order MUST be defined
+by at least a human-readable extension specification and, preferably, a
+machine-readable specification.
+            </li>
+            <li>
+In addition to the rule above, JSON-LD-based processors MUST produce an error
+when a JSON-LD Context redefines any term in the
+<a href="https://www.w3.org/TR/json-ld/#dfn-active-context">active context</a>.
+            </li>
+          </ul>
+
+          <p>
+To avoid the possibility of accidentally overriding terms, developers are urged
+to scope their extensions. For example, the following extension scopes the
+new <code>favoriteFood</code> term so that it may only be used within the
+<code>claim</code> property:
```

Comment:
This may count as a "redefinition" of the `claim` term. I think we should still merge this PR as it gives us a starting point for tweaking these rules -- but we should take note of this.